ballerina/Ballerina.toml

```diff
@@ -5,12 +5,12 @@ name = "hubspot.marketing.emails"
 version = "0.1.0"
 license = ["Apache-2.0"]
 authors = ["Ballerina"]
-keywords = []
-# icon = "icon.png" # TODO: update icon.png
+keywords = [] # TODO: Add keywords
```

Comment:
Build the package again locally to update the keywords. They should be automatically added here when you update them in the `Ballerina.toml` file inside the `build-config` directory and build the connector again.